### DISCUSSION TITLE
a query about exporting notes from ASReview

### DISCUSSION THREAD

**Discussion by BobbWANG** [asreview/asreview#1415] [Q&A] [answered]
We are currently screening articles using it for a meta-analysis. We found that some articles have no DOIs and no abstracts, so we first labeled them as relevant and left notes on those articles for further screening. However, we found that the exported file from ASReview didn't include the notes we left during screening in any of the columns. We tried different file formats and also deleting the "notes" column before uploading to ASReview in case the column name conflicted with the one in the exported file, but none of them worked. 

Solution: I then found out that it was because my release is not up to date. After running "pip install --upgrade asreview" in the terminal, the problem was solved. To help others who may have similar questions, I'm posting this question here and answering it myself! 

Thanks again for creating and sharing this wonderful tool!

**Comment by Rensvandeschoot** [comment, marked as answer]
Indeed, the option to export notes has been contributed by @rohitgarud in version 1.2!!
